pmd-java/src/test/java/net/sourceforge/pmd/lang/java/ast/ParserCornersTest.java

```diff
@@ -276,7 +280,7 @@ public void testMethodReferenceConfused() throws Exception {
     private String readAsString(String resource) {
         InputStream in = ParserCornersTest.class.getResourceAsStream(resource);
         try {
-            return IOUtils.toString(in);
+            return IOUtils.toString(in, StandardCharsets.UTF_8);
```

Comment:
While we are here fixing this IOUtils.toString() call, we should also fix the following IOUtils.closeQuietly(in) call. Simply replacing this try-statement with a try-with-resources statement makes the call to closeQuietly obsolete.

Reply:
I was planning to replace `IOUtils.closeQuietly(in)` in a separate commit and keep this one focused on Charsets. Is that okay? Or do you think I should roll that into this PR?

Reply:
That's fine! :+1: